src/lib/field/_core.layout.scss

```diff
@@ -113,122 +138,83 @@
 
 @mixin support-text-inset($inset-direction: none) {
   @if $inset-direction == 'both' or $inset-direction == 'start' {
-    padding-inline-start: #{token(container-padding-inline-start)};
+    @include item-padding(start, outer);
   }
-
+  
   @if $inset-direction == 'both' or $inset-direction == 'end' {
-    padding-inline-end: #{token(container-padding-inline-start)};
+    @include item-padding(end, outer);
   }
 }
 
 //
 // Multiline
 //
 
-$resize-container-input-offset-base: calc((token('height', custom) - typography.variable(body2, line-height)) / 2);
-$resize-container-input-padding-block-end-base: 8px;
-
-@mixin container-multiline {
-  display: block;
-  block-size: fit-content;
-}
+$multiline-input-offset-base: calc((token(height, custom) - typography.variable(body2, line-height)) / 2);
+$multiline-input-padding-block-end-base: 8px;
 
-@mixin resize-container {
+@mixin multiline-container {
   align-items: start;
 
-  border-radius: inherit;
-
+  block-size: fit-content;
   min-inline-size: #{token(multiline-min-inline-size)};
   max-inline-size: #{token(multiline-max-inline-size)};
   min-block-size: #{token(multiline-min-block-size)};
   max-block-size: #{token(multiline-max-block-size)};
 
-  overflow: hidden;
   resize: #{token(multiline-resize)};
 }
 
-@mixin resize-container-inset-label {
+@mixin multiline-container-child {
+  align-items: center;
+  block-size: #{token(height)};
+}
+
+@mixin multiline-accessory-slot {
+  @include multiline-container-child;
+
+  display: flex;
+  align-self: start;
+}
+
+@mixin multiline-inset-label {
   pointer-events: none;
   inline-size: fit-content;
 }
 
-@mixin resize-container-inset-label-slot {
+@mixin multiline-inset-label-slot {
   display: block;
 }
 
-@mixin resize-container-inset-label-background {
-    @include floating-animation(resize-container-inset-label-background-animation);
+@mixin multiline-inset-label-background {
+    @include animation.floating-animation(multiline-inset-label-background-animation);
     
     position: absolute;
     z-index: -1;
-    inset-inline-start: #{token(border-width)};
+    inset-inline-start: #{token(outline-width)};
 
-    transition-duration: #{token(container-animation-duration)};
+    transition-duration: #{token(surface-animation-duration)};
     transition-property: background;
-    transition-timing-function: #{token(container-animation-timing)};
+    transition-timing-function: #{token(surface-animation-timing)};
     
-    background: #{token(resize-container-inset-label-background, custom)};
+    background: #{token(multiline-inset-label-background, custom)};
     
     inline-size: 100%;
-    block-size: typography.variable(body2, line-height);
+    block-size: 3ex;
```

Comment:
Just curious about the usage of `ex` units here

Reply:
The height of the background here doesn't need to be precise so for simplicity I just set it to three times the x height of the font 🤷‍♂️